docs/book/component-gallery/orchestrators/airflow.md

```diff
@@ -58,12 +74,52 @@ provision it. When it is finished, it will print a
 username and password which you can use to log in to the Airflow UI
 [here](http://0.0.0.0:8080).
 
+As long as you didn't configure any custom value for the `dag_output_dir`
+attribute of your orchestrator, running a pipeline locally is as simple 
+as calling:
 
-You can now run any ZenML pipeline using the Airflow orchestrator:
 ```shell
 python file_that_runs_a_zenml_pipeline.py
 ```
 
+This call will write a `.zip` file containing a representation of your ZenML
+pipeline to the Airflow DAGs directory. From there, the local Airflow server
+will load it and run your pipeline (It might take a few seconds until the pipeline
+shows up in the Airflow UI).
+{% endtab %}
+
+{% tab title="Remote" %}
+
+When using the Airflow orchestrator with a remote deployment, you'll additionally 
+need:
+* A remote ZenML server deployed to the cloud. See the [deployment guide](../../getting-started/deploying-zenml/deploying-zenml.md) for more information.
+* A deployed Airflow server. See the [deployment section](#how-to-deploy-it) 
+for more information.
+* A [remote artifact store](../artifact-stores/artifact-stores.md) as part of 
+your stack.
+* A [remote container registry](../container-registries/container-registries.md) 
+as part of your stack.
```

Comment:
Just as an idea, it would be really cool to have a diagram to show how these 'things' will interact.

Reply:
i guess out of scope for this ticket, but in general something we should do - but yes exactly how the zenserver artifact store and orchestrator interact within a pipeline run